### ISSUE TITLE
Access Plans creates multiple llms-access-plan-description and llms-access-plan-footer

### ISSUE THREAD

**Issue by AndreaBarghigiani** [gocodebox/lifterlms#639]
### 1. Reproduction Steps

I am working with a new project where I notice something different on how LifterLMS is showing the pricing table and I notice this after upgrading to v3.23.0 and is something that I notice only on this website. 

Each time there is an access plan in a course, for [example here](http://33422.s.time4vps.cloud/cannascienza/corso/sistema-endocannabinoide/), LifterLMS adds a coupple of `llms-access-plan-description` and `llms-access-plan-footer` as showed in the image below.

![lifterlms-strange-behaviour](https://user-images.githubusercontent.com/190159/45355192-0e1e1180-b5c0-11e8-98dd-3599043bafdb.jpg)

This wouldn't be a problem if the empty `llms-access-plan-footer` inside the first `llms-access-plan-description` didn't apply some paddings.

![schermata 2018-09-11 alle 12 41 21](https://user-images.githubusercontent.com/190159/45355849-d748fb00-b5c1-11e8-9835-494ee2f9bef5.jpg)

I know that I could fix it with a bit of CSS but I would like to undestand why this is happening. I am usign [Simply Show Hook](https://wordpress.org/plugins/simply-show-hooks/) and I don't notice anything strange there.

I've deactivated all the plugins and also the `single-course.php` custom page that I've created but the problem persists. [Here](https://gist.github.com/AndreaBarghigiani/e00df4e1fc6b9de2d17abae7c8e5b9fd) you can see the code of this page if you want to check. 

I've also tried another theme but the problem persists.

Do you have any idea why is this happening?

### 2. Expected Behavior

Showing the Access Plan without duplicated or empty elements.

### 3. Actual Behavior

The `llms-access-plan-description` and `llms-access-plan-footer` are duplicated and the last one result empty inside the first `llms-access-plan-description`.

### 4. Error Messages / Logs

No error messages.

### 5. System Report

Wordpress
-------------------------------------------

Home Url: http://0.0.0.0:8080
Site Url: http://0.0.0.0:8080
Version: 4.9.8
Debug Mode: No
Locale: it_IT
Multisite: No
Page For Posts: Not Set
Page On Front: Not Set
Permalink Structure: /%postname%/
Show On Front: posts
Wp Cron: Yes

Settings
-------------------------------------------

Version: 3.23.0
Db Version: 3.23.0
Course Catalog: Tutti i corsi (#5)
Course Catalog Per Page: 9
Course Catalog Sorting: menu_order,ASC
Membership Catalog: Membership Catalog (#6)
Membership Catalog Per Page: 9
Membership Catalog Sorting: menu_order
Site Membership: Not Set
Student Dashboard: Bacheca (#8)
Courses Endpoint: miei-corsi
Edit Endpoint: modifica-account
Lost Password Endpoint: password-persa
Vouchers Endpoint: riscatta-voucher
Autogenerate Username: yes
Password Strength Meter: yes
Minimum Password Strength: strong
Terms Required: no
Terms Page: Not Set
Checkout Names: required
Checkout Address: required
Checkout Phone: optional
Checkout Email Confirmation: yes
Open Registration: no
Registration Names: required
Registration Address: optional
Registration Phone: hidden
Registration Voucher: optional
Registration Email Confirmation: no
Account Names: required
Account Address: required
Account Phone: optional
Account Email Confirmation: yes
Checkout Page: Purchase (#7)
Confirmation Endpoint: confirm-payment
Force Ssl Checkout: no
Country: IT
Currency: EUR
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Decimals: 2
Trim Zero Decimals: no
Recurring Payments: no
Email From Address: andrea@codeat.it
Email From Name: Cannabiscienza
Email Footer Text: 
Email Header Image: 
Cert Bg Width: 800
Cert Bg Height: 616
Cert Legacy Compat: no

Gateways
-------------------------------------------

Manuale: Enabled
Manuale Logging: no
Manuale Order: 1

Server
-------------------------------------------

Mysql Version: 5.5.5
Php Curl: Yes
Php Default Timezone: UTC
Php Fsockopen: Yes
Php Max Input Vars: 1000
Php Max Upload Size: 2 MB
Php Memory Limit: 256M
Php Post Max Size: 8M
Php Soap: No
Php Suhosin: No
Php Time Limt: 30
Php Version: 7.2.6
Software: Apache/2.4.25 (Debian)
Wp Memory Limit: 40M

Browser
-------------------------------------------

HTTP USER AGENT: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.13; rv:63.0) Gecko/20100101 Firefox/63.0

Theme
-------------------------------------------

Name: Cannabiscienza Theme
Version: 2.6.0
Themeuri: https://cannabiscienza.it/
Authoruri: https://codeat.it/
Template: genesis
Child Theme: Yes
Llms Support: Yes

Plugins
-------------------------------------------

Cannabiscienza: 1.0.0
Genesis eNews Extended: 2.1.1
Genesis Simple FAQ: 0.9.1
Gutenberg: 3.7.0
LifterLMS: 3.23.0
Simple Social Icons: 2.0.1
Simply Show Hooks: 1.2.1
WP Job Manager: 1.31.3

Integrations
-------------------------------------------

BbPress: No
BuddyPress: No

Template Overrides
-------------------------------------------

archive-course.php (ver: 3.0.0): /var/www/html/wp-content/themes/cannascienza-theme/lifterlms/ (ver: 3.0.0)
course/author.php (ver: 3.13.0): /var/www/html/wp-content/themes/cannascienza-theme/lifterlms/ (ver: 3.13.0)
course/categories.php (ver: ): /var/www/html/wp-content/themes/cannascienza-theme/lifterlms/ (ver: )
course/lesson-navigation.php (ver: ): /var/www/html/wp-content/themes/cannascienza-theme/lifterlms/ (ver: )
course/lesson-preview.php (ver: 3.19.2): /var/www/html/wp-content/themes/cannascienza-theme/lifterlms/ (ver: 3.19.2)
course/outline-list-small.php (ver: 3.19.2): /var/www/html/wp-content/themes/cannascienza-theme/lifterlms/ (ver: 3.19.2)
course/parent-course.php (ver: 3.0.0): /var/www/html/wp-content/themes/cannascienza-theme/lifterlms/ (ver: 3.0.0)
course/syllabus.php (ver: 3.0.0 - refactored for sanity's sake): /var/www/html/wp-content/themes/cannascienza-theme/lifterlms/ (ver: 3.0.0 - refactored for sanity's sake)
course/tracks.php (ver: ): /var/www/html/wp-content/themes/cannascienza-theme/lifterlms/ (ver: )
myaccount/dashboard-section.php (ver: 3.19.5): /var/www/html/wp-content/themes/cannascienza-theme/lifterlms/ (ver: 3.19.5)
product/pricing-table.php (ver: 3.23.0): /var/www/html/wp-content/themes/cannascienza-theme/lifterlms/ (ver: 3.23.0)

### 6. Browser, Device, and Operating System

It happens with any browser.

**Comment by thomasplevy** [comment]
@AndreaBarghigiani can you see if this is reproducible with a twenty theme? My local install does not show this bug and I see you have a number of template overrides in your theme so I'm wondering if that's accidentally causing the duplicates?

Thanks,

**Comment by AndreaBarghigiani** [comment]
Hi @thomasplevy, 
I've already tried with a Twnty Seventeen theme and the problem persists but *I've found the problem* and it was a fault of mine that I could see only checking the _Text View_ of the access plan.

![schermata 2018-09-12 alle 08 08 47](https://user-images.githubusercontent.com/190159/45405464-3dd22580-b663-11e8-93f3-c415a4b2ba98.jpg)

I am terribly sorry if I didn't check it before but I couldn't image that the copy/paste I did it brought the HTML elements as well.

Sorry if I wasted your time and keep up your amazing work!
